package.json

```
@@ -169,142 +157,88 @@
                 "category": "Azure Container Apps"
             }
         ],
-        "views": {
-            "azure": [
-                {
-                    "id": "containerApps",
-                    "name": "Container Apps"
-                }
-            ]
-        },
         "menus": {
-            "view/title": [
-                {
-                    "command": "containerApps.createContainerApp",
-                    "when": "view == containerApps",
-                    "group": "navigation@1"
-                },
-                {
-                    "command": "containerApps.refresh",
-                    "when": "view == containerApps",
-                    "group": "navigation@2"
-                }
-            ],
             "view/item/context": [
-                {
-                    "command": "containerApps.selectSubscriptions",
-                    "when": "view == containerApps && viewItem == azureextensionui.azureSubscription",
-                    "group": "inline"
-                },
-                {
-                    "command": "containerApps.refresh",
-                    "when": "view == containerApps && viewItem =~ /azureSubscription|managedEnvironment|containerApp/i",
-                    "group": "z@2"
-                },
                 {
                     "command": "containerApps.createManagedEnvironment",
-                    "when": "view == containerApps && viewItem =~ /azureSubscription/i",
+                    "when": "view == azureResourceGroups && viewItem =~ /azureResourceTypeGroup.*microsoft.app/managedenvironments/",
```

I think this is a safer bet:
```suggestion
                    "when": "view == azureResourceGroups && viewItem =~ /azureResourceTypeGroup/i && viewItem =~ /microsoft.app/managedenvironments/i",
```
(And maybe not `i` depending on the below comment thread)